config/ui-extensions/apigateways/form

```diff
@@ -0,0 +1,12 @@
+- path: spec
+  simple: true
+  name: General
+  widget: FormGroup
+  defaultExpanded: true
+  children:
+  - path: enableKymaGateway
+    simple: true
+    name: enableKymaGateway
+    inputInfo: enableKymaGateway
```

Comment:
perhaps without `inputInfo` because is kind of duplicating?:
![image](https://github.com/kyma-project/api-gateway/assets/10164673/b44aae46-08bc-4c82-ad25-a9c9028d614f)